### ISSUE TITLE
test t/public_key.t fails: reason given is NODATA

### ISSUE THREAD

**Issue by ncopa** [marcbradshaw/mail-dkim#9]
```
t/Mail-DKIM.t ................... ok   
t/policy.t ...................... ok     
t/public_key.t .................. 1/5 
#   Failed test 'reason given is NODATA'
#   at t/public_key.t line 36.
# Looks like you failed 1 test of 5.
t/public_key.t .................. Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/5 subtests 
        (less 2 skipped subtests: 2 okay)
```

Seems like the response it gets is NXDOMAIN, which I guess is correct for a non-existing domain?

**Comment by marcbradshaw** [comment]
Looks like the response has changed, I'll update the test and push a new version

**Comment by marcbradshaw** [comment]
Fixed in v1.20200724, thanks for the report.

**Comment by ncopa** [comment]
Thank you for quickest response and new release I have ever seen! :1st_place_medal:
